cli/src/cli.rs

```diff
@@ -0,0 +1,97 @@
+use clap::{Parser, Subcommand};
+
+/**
+Welcome to the Simperby CLI!
+*/
+#[derive(Debug, Parser)]
+#[clap(name = "git")]
+#[clap(about = "A Simperby client CLI", long_about = None)]
+pub struct Cli {
+    #[clap(subcommand)]
+    command: Commands,
+}
+
+#[derive(Debug, Subcommand)]
+pub enum Commands {
+    /// Initialize a new Simperby node in the current directory.
+    Init,
+    /// Sync the `main` branch to the given commit.
+    ///
+    /// This will verify every commit along the way.
+    /// If the given commit is not a descendant of the
+    /// current `main` (i.e., cannot be fast-forwarded), it fails.
+    ///
+    /// Note that if you sync to a block `H`, then the `main` branch will move to `H-1`.
+    /// To sync the last block `H`, you have to run `update`.
+    /// (This is because that the finalization proof for a block appears in the next block.)
+    Sync { commit: String },
+    /// Print the information about the block data Git server that this node is hosting.
+    Git,
+    /// Clean the repository, removing all the outdated (incompatible with `main`) branches.
+    Clean {
+        /// If enabled, it will remove all the branches except `main`.
+        #[clap(long, action)]
+        hard: bool,
+    },
+    /// Vote on the agenda, broadcasting to the network.
+    /// It will also leave a `vote` tag on the commit (with some postfix).
+    Vote { commit: String },
+    /// Propose the given agenda or block.
+    ///
+    /// 1. If the commit is the last transaction,
```

Comment:
What's the meaning of "last"? What transaction is the "last transaction"?

Reply:
Updated

Reply:
How can I notify the CLI that the transaction I'm committing is the last transaction of an agenda?

Reply:
By specifying the commit, as in the only argument of `Vote`. Note that a transactions is also a commit.

Reply:
ok. I got it.